src/test/java/com/commafeed/MockingTest/RedisCacheServiceMockingTest.java

```diff
@@ -0,0 +1,47 @@
+package com.commafeed.MockingTest;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import org.junit.Before;
+import org.junit.Test;
+import redis.clients.jedis.Jedis;
+import redis.clients.jedis.JedisPool;
+
+import com.commafeed.backend.cache.RedisCacheService;
+import com.commafeed.backend.model.Feed;
+
+import static org.mockito.Mockito.*;
+import static org.junit.Assert.*;
+
+public class RedisCacheServiceMockingTest {
+	JedisPool mockpool;
+	Feed feedMock;
+	Jedis j;
+	Set<String> members;
+	@Before
+	public void setUp() { 
+		mockpool = mock(JedisPool.class);
+		feedMock = mock(Feed.class);
+		j = mock(Jedis.class);
+		members = new HashSet<String>();
+        members.add("Chris");
+	}
+	
+	
+	@Test
+    public void testGetLastEntries() {
+       
+        when(mockpool.getResource()).thenReturn(j);
+        when(feedMock.getId()).thenReturn(1L);
+        when(j.smembers("f:1")).thenReturn(members);
+        RedisCacheService rcs = new RedisCacheService(mockpool);
+
+        List<String> tempList = new  ArrayList<>();
+        tempList = rcs.getLastEntries(feedMock);
+        System.out.println(tempList.get(0));
```

Comment:
You should remove the System.out.println after testing that your test behave as expected. If you don't every time the test will be run (when building or anything else), it will output a single list of entries.